src/sentry/static/sentry/app/components/dropdownAutoComplete.jsx

```diff
@@ -0,0 +1,165 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'react-emotion';
+import _ from 'lodash';
+import AutoComplete from './autoComplete';
+import Input from '../views/settings/components/forms/controls/input';
+
+class DropdownAutoComplete extends React.Component {
+  static propTypes = {
+    items: PropTypes.array,
```

Comment:
items is a complicated enough proptype that you may want to elaborate its shape further than just array. something along the lines of 
PropTypes .oneOfType([
      // flat
      PropTypes.arrayOf(PropTypes. shape({
         value: string ,
         display: string  
      }),
      // groups
      PropTypes.arrayOf(PropTypes. shape({
           group stuff 
           items:
      })
])